src/actions/teamActions.js

```diff
@@ -0,0 +1,24 @@
+import { LOAD_TEAMS_SUCCESS, TOGGLE_TEAM } from './actionTypes';
+import teamApi from '../api/teamApi';
+
+export function loadTeams() {
+  return function(dispatch) {
+    return teamApi.getAllTeams().then(teams => {
+      const teamsArray = teams.map(team => ({
+        ...team, visible: true
+      }));
+      dispatch(loadTeamsSuccess(teamsArray));
+    }).catch(error => {
+      throw(error);
+    });
+  };
+}
+
+export function loadTeamsSuccess(teams) {
+  return { type: LOAD_TEAMS_SUCCESS, teams: teams };
+}
+
+export const toggleTeam = team_name => ({
+  type: TOGGLE_TEAM,
+  team_name,
```

Comment:
I'm not sure when I should use `camelCase` and when `underscore_variable`. Can you please clarify it for me?